red/runtime/nodes/flows/index.js

```
@@ -124,12 +263,93 @@ function setFlows(_config,type,muteLog,forceStart) {
+            diff = { added: [], changed: [], removed: [], rewired: [] }; // Empty diff
+        } else if (batchOptimization && activeFlowConfig) {
+            var cachedDiff = nodeCache.getCachedDiff(activeFlowConfig, newFlowConfig);
+            if (cachedDiff) {
+                diff = cachedDiff;
+            } else {
+                diff = nodeCache.computeIncrementalDiff(activeFlowConfig, newFlowConfig) || 
+                       flowUtil.diffConfigs(activeFlowConfig,newFlowConfig);
+                nodeCache.cacheDiff(activeFlowConfig, newFlowConfig, diff);
+            }
+        } else {
+            var cachedDiff = nodeCache.getCachedDiff(activeFlowConfig, newFlowConfig);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'cachedDiff'. Consider to delete it or rename it.

'cachedDiff' is defined here:


(lint/suspicious/noRedeclare)